nixos/modules/misc/locate.nix

```diff
@@ -147,8 +157,9 @@ in {
         serviceConfig.PrivateTmp = "yes";
         serviceConfig.PrivateNetwork = "yes";
         serviceConfig.NoNewPrivileges = "yes";
-        serviceConfig.ReadOnlyDirectories = "/";
-        serviceConfig.ReadWriteDirectories = dirOf cfg.output;
+        serviceConfig.ReadOnlyPaths = "/";
+        # TODO: one could try to further reduce this to cfg.output by pre-creating the file in locatedb-setup
+        serviceConfig.ReadWritePaths = dirOf cfg.output;
```

Comment:
`ConditionPathExists=` might also lead to better error messages.

Reply:
Just tested it, sadly neither `ConditionPathExists` nor `AssertPathExists` seem to have any effect, it's always just the `NAMESPACE` error. 😞